game/script-greetings.rpy

```diff
@@ -334,6 +338,24 @@ label greeting_leave_return:
 
     return
 
+label greeting_restart:
+    $ jn_player_restarted = False
+    n 1nnmsm "Hey,{w=0.1} there you are!"
+    show natsuki 4nnmpu
+    menu:
+        n "Did the restart go okay?"
+
+        "Yes, it went well.":
+
+            n 2nchgn "Good!"
+
+        "No, it didn't work.":
+
+            n 1nnmpu "Oh,{w=0.1} that's a shame."
+            n "Well,{w=0.2}{nw}"
+            extend 4nlraj " if you need to restart again,{w=0.1} just let me know okay?"
+    return
+
```

Comment:
Putting aside the restart aspect feeling a bit out of place, this dialogue raises some eyebrows: if the restart _didn't_ succeed, I wouldn't imagine Natsuki and the player would be having this conversation!

It also seems too sudden in how the conversation is resolved: I would personally expect Natsuki to tease the player a little (if they're close), and/or at least welcome the player formally on conclusion/ask what's up in line with the standard greetings. As it stands, Natsuki just saying _"Good!"_ feels a little lacklustre to me.

If a "Be right back" theme is taken, then we could also take this opportunity to grill the player if they kept Natsuki waiting too long, or just failed to turn up at all! (Like with `greeting_leave_return`).